tables/write-manuscript-tables.nb.html

```diff
@@ -3205,7 +3205,7 @@ <h2>Table 2: Patients with hypermutant or ultra-hypermutant tumors</h2>
 <!-- rnb-source-end -->
 <div data-pagedtable="false">
 <script data-pagedtable-source type="application/json">
-{"columns":[{"label":["Kids_First_Biospecimen_ID"],"name":[1],"type":["chr"],"align":["left"]},{"label":["cancer_group"],"name":[2],"type":["chr"],"align":["left"]},{"label":["short_histology"],"name":[3],"type":["chr"],"align":["left"]}],"data":[{"1":"BS_20TBZG09","2":"High-grade glioma astrocytoma","3":"HGAT"},{"1":"BS_8AY2GM4G","2":"High-grade glioma astrocytoma","3":"HGAT"},{"1":"BS_P3PF53V8","2":"High-grade glioma astrocytoma","3":"HGAT"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
+{"columns":[{"label":["Kids_First_Biospecimen_ID"],"name":[1],"type":["chr"],"align":["left"]},{"label":["cancer_group"],"name":[2],"type":["chr"],"align":["left"]},{"label":["short_histology"],"name":[3],"type":["chr"],"align":["left"]}],"data":[{"1":"BS_02YBZSBY","2":"High-grade glioma astrocytoma","3":"HGAT"},{"1":"BS_20TBZG09","2":"High-grade glioma astrocytoma","3":"HGAT"},{"1":"BS_8AY2GM4G","2":"High-grade glioma astrocytoma","3":"HGAT"},{"1":"BS_F0GNWEJJ","2":"Neuroblastoma","3":"Neuroblastoma"},{"1":"BS_HM5GFJN8","2":"Diffuse midline glioma","3":"HGAT"},{"1":"BS_P3PF53V8","2":"High-grade glioma astrocytoma","3":"HGAT"},{"1":"BS_VW4XN9Y7","2":"High-grade glioma astrocytoma","3":"HGAT"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
```

Comment:
Why did this change? This is a real output change, so it has my curiousity piqued.

Reply:
never mind... this was a change in an intermediate diff... maybe you were testing with a limited dataset.

Reply:
Yes, there had been some testing locally, which is all cleaned up now! The output rendered HTML is the same as what's currently in `master`.
Just for posterity:

## Currently in `master`
> https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/83545e5b38f9a71db376c3c1d34f5cb01e3e2bfe/tables/output_tables.nb.html
<img width="849" alt="Screenshot 2023-03-21 at 3 33 48 PM" src="https://user-images.githubusercontent.com/4701111/226721172-62137f96-c971-4ffc-a1c3-eb3cfe7144e1.png">


## This PR
<img width="866" alt="Screenshot 2023-03-21 at 3 34 03 PM" src="https://user-images.githubusercontent.com/4701111/226721225-8b9b5206-580e-4efb-8fc9-847e84969080.png">